.github/workflows/test-linux.yml

```
@@ -0,0 +1,40 @@
+name: Unit-tests on Linux
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+          - "3.9"
+          - "3.10"
```

```suggestion
          - "3.10"
          - "3.11"
```

would replace #7155 @osalpekar.

---

Tested this in the commits starting with 62dd31aff7babb2de6305257960849f297689e35 and works out fine.

---

Sounds great - thanks!